### PR TITLE
Add courses metadata

### PR DESCRIPTION
**Pull request by jgagne**
## What this PR does

- Adds missing courses metadata and assets

#### Meta

- https://deploy-preview-758--thegymcms.netlify.app/courses/full/gym-101/meta/
- https://deploy-preview-758--thegymcms.netlify.app/courses/full/gym-105/meta/
- https://deploy-preview-758--thegymcms.netlify.app/courses/gym-shorts/gym-002/meta/
- https://deploy-preview-758--thegymcms.netlify.app/courses/gym-shorts/gym-003/meta/

#### Announcement

- https://deploy-preview-758--thegymcms.netlify.app/courses/full/gym-101/announcement/
- https://deploy-preview-758--thegymcms.netlify.app/courses/full/gym-105/announcement/
- https://deploy-preview-758--thegymcms.netlify.app/courses/gym-shorts/gym-002/announcement/
- https://deploy-preview-758--thegymcms.netlify.app/courses/gym-shorts/gym-003/announcement/

### CODE REVIEW

**Comment by rediris** [comment]
@jgagne we should add one for GYM-101 as well: https://github.com/gymnasium/studio-GYM-101

**Comment by rediris** [comment]
@jgagne we should add one for GYM-101 as well: https://github.com/gymnasium/studio-GYM-101

**Comment by jgagne** [comment]
@rediris [Totally!](https://github.com/gymnasium/teamwork/projects/6) (Coming soon.) Thanks for setting up the Studio repo!

**Comment by rediris** [comment]
> @rediris [Totally!](https://github.com/gymnasium/teamwork/projects/6) (Coming soon.) Thanks for setting up the Studio repo!

No problem! I'm starting to work on the studio repo for 003 as well. Coming up next!

File: _data/courses/GYM-002.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+course_ID: GYM-002
+title: "Introducing Node.js"
+date: 2022-05-27T00:00:00-04:00
+course_type: short
+live: false
+retired_message:
+  1: "<strong>This course has been retired.</strong> Due to outdated content, we dont offer a badge for this course  but you can still view the lessons."
```
Comment (rediris) [inline _data/courses/GYM-002.yml]: add an apostrophe for `don't`?

Comment (rediris) [reply _data/courses/GYM-002.yml]: Or a fancy apostrophe? `don’t`

Comment (jgagne) [reply _data/courses/GYM-002.yml]: Thanks for the extra 👀 on this!

Comment (rediris) [reply _data/courses/GYM-002.yml]: No problem! If you're still taking requests, there appears to be an extra space between `course  but`…

**Review by rediris — APPROVED** [review]

**Pull request merged**